cvcl/models.py

```diff
@@ -284,3 +284,58 @@ class Meta:
 
     def __str__(self):
         return str(self.id)
+
+
+class InstallPackage(models.Model):
+    name = models.CharField(max_length=30)
+    package_update = models.BooleanField(default=False)
+    package_upgrade = models.BooleanField(default=False)
+    package_reboot_if_required = models.BooleanField(default=False)
+    vm_definition = models.ForeignKey('VmDefinition', on_delete=models.CASCADE, null=True,
+                                      related_name='install_packages')
+
+    class Meta:
+        verbose_name = "Install Packages"
+
+    def __str__(self):
+        return str(self.id)
+
+
+class AddUser(models.Model):
```

Comment:
Not really, Model is a singular thing, must have a singular name. `Package` could work.